### PR TITLE
Adds onboarding screen for analyses

### PR DESCRIPTION
**Pull request by javierarce**
This PR adds a new kind of modal window that appears after [certain analyses](https://github.com/CartoDB/cartodb/blob/7d69ea7ddb83262e93a82211a942b860f54c72b4/lib/assets/javascripts/cartodb3/components/onboardings/analysis/analyses-types.js) are finished, showing useful information on what an analysis has just done to the user's dataset.

Users can close the window, go to the style editor and check an input field to avoid been notified again. For this last feature, I created [a new class](https://github.com/CartoDB/cartodb/blob/7d69ea7ddb83262e93a82211a942b860f54c72b4/lib/assets/javascripts/cartodb3/components/local-storage/local-storage.js) to manage the native localStorage class.

You can follow the drafting of the texts that appear in those screens from [this ticket](https://github.com/CartoDB/cartodb/issues/9034).

screen shot 2016-07-21 at 17 59 18

screen shot 2016-07-21 at 18 09 51

### CODE REVIEW

**Comment by javierarce** [comment]
Right now the "Never show me this message again" is linked to the type of analysis, so closing a georeference analysis for a particular layer will prevent showing the screen in other analyses. Should this work like this, @urbanphes @saleiva?

File: Gemfile.lock [hunk 1]
```diff
@@ -422,6 +422,3 @@ DEPENDENCIES
   virtus (= 1.0.5)
   vizzuality-sequel-rails (= 0.3.7)!
   webrick (= 1.3.1)
-
-BUNDLED WITH
```
Comment (xavijam) [inline Gemfile.lock]: ouch!

**Comment by xavijam** [comment]
Fine by me! just take into account tests and the gemfile thing :)

**Comment by xavijam** [comment]
:es:

**Pull request merged**